server/data/elite2ClientBuilder.js

```diff
@@ -49,5 +49,16 @@ module.exports = token => {
             : logger.warn(error, `Error calling elite2`),
       })
     },
+    getActiveAgenciesByType(type) {
+      return restClient.get({
+        path: `/api/agencies/type/${type}`,
+      })
+    },
+
+    getPrisonById(agencyId) {
+      return restClient.get({
+        path: `/api/agencies/prison/${agencyId}`,
```

Comment:
Think it might be better to use `/api/agencies/{agencyId}?activeOnly=false`

It returns the least amount of data that you require - for instance, doesn't rely on contact details and is a bit more explicit around that you would want to potentially return inactive prisons too.

Reply:
In fact, its a good idea to check to see what the API will actually return - looking at the elite2 codebase.

It seems in this case the path of the url suggests it's returning prisons but its really all about serving a use case around retrieving contact details - as such it doesn't make sense to return inactive prisons if you are only interested in contact details. 

If you track through the api call, you'll see it eventually leads down to a query called `FIND_PRISON_ADDRESSES_PHONE_NUMBERS` and this explicitly hard codes the active flag:   
`WHERE al.ACTIVE_FLAG = 'Y'`

Reply:
hmmm.... have changed the url to include ?activeOnly=false but it hasn't made a difference in terms of returning the prison details from its id ...

Reply:
done